src/js/actions/ActionsCreators.js

```diff
@@ -280,8 +279,11 @@ export function setPayout(payload) {
     if (typeof payout === 'undefined') {
       payout = getState().getIn(['storage', 'payout']);
     }
+    else if (isNaN(payout*1) || !Number.isInteger(payout*1) || parseInt(payout) < 0 || parseInt(payout) > 100) {
```

Comment:
@qingweibinary we are checking typeof so for null value (when user backspace to empty) we can reset to 1, but for empty value (first reload) we can read the value from the last time they stored